muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/reducers/main-function/guider/index.ts

```diff
@@ -120,91 +119,91 @@ export default function guider( state: GuiderType = {
+}, action: ActionType): GuiderType {
+  if (action.type === "LOCK_TOOLBAR") {
     return Object.assign({}, state, {
       toolbarLock: true
     });
-  } else if (action.type === "UNLOCK_TOOLBAR"){
+  } else if (action.type === "UNLOCK_TOOLBAR") {
     return Object.assign({}, state, {
       toolbarLock: false
     });
-  } else if (action.type === "UPDATE_GUIDER_ACTIVE_FILTERS"){
+  } else if (action.type === "UPDATE_GUIDER_ACTIVE_FILTERS") {
     return Object.assign({}, state, {
       activeFilters: action.payload
     });
-  } else if (action.type === "UPDATE_GUIDER_ALL_PROPS"){
-    return Object.assign( {}, state, action.payload );
-  } else if (action.type === "UPDATE_GUIDER_STATE"){
-    return Object.assign( {}, state, {
+  } else if (action.type === "UPDATE_GUIDER_ALL_PROPS") {
+    return Object.assign({}, state, action.payload);
+  } else if (action.type === "UPDATE_GUIDER_STATE") {
+    return Object.assign({}, state, {
       state: action.payload
     });
-  } else if (action.type === "ADD_TO_GUIDER_SELECTED_STUDENTS"){
+  } else if (action.type === "ADD_TO_GUIDER_SELECTED_STUDENTS") {
     let student: GuiderStudentType = action.payload;
-    return Object.assign( {}, state, {
-      selectedStudents: state.selectedStudents.concat( [student] ),
-      selectedStudentsIds: state.selectedStudentsIds.concat( [student.id] )
+    return Object.assign({}, state, {
+      selectedStudents: state.selectedStudents.concat([student]),
+      selectedStudentsIds: state.selectedStudentsIds.concat([student.id])
     });
-  } else if (action.type === "REMOVE_FROM_GUIDER_SELECTED_STUDENTS"){
+  } else if (action.type === "REMOVE_FROM_GUIDER_SELECTED_STUDENTS") {
     let student: GuiderStudentType = action.payload;
-    return Object.assign( {}, state, {
-      selectedStudents: state.selectedStudents.filter( s => s.id !== student.id ),
-      selectedStudentsIds: state.selectedStudentsIds.filter( id => id !== student.id )
+    return Object.assign({}, state, {
+      selectedStudents: state.selectedStudents.filter(s => s.id !== student.id),
+      selectedStudentsIds: state.selectedStudentsIds.filter(id => id !== student.id)
     });
-  } else if (action.type === "SET_CURRENT_GUIDER_STUDENT"){
-    return Object.assign( {}, state, {
+  } else if (action.type === "SET_CURRENT_GUIDER_STUDENT") {
+    return Object.assign({}, state, {
       currentStudent: action.payload
     });
-  } else if (action.type === "SET_CURRENT_GUIDER_STUDENT_EMPTY_LOAD"){
-    return Object.assign( {}, state, {
+  } else if (action.type === "SET_CURRENT_GUIDER_STUDENT_EMPTY_LOAD") {
+    return Object.assign({}, state, {
       currentStudent: {},
       currentState: "LOADING"
     });
-  } else if (action.type === "SET_CURRENT_GUIDER_STUDENT_PROP"){
+  } else if (action.type === "SET_CURRENT_GUIDER_STUDENT_PROP") {
     let obj: any = {};
     obj[action.payload.property] = action.payload.value;
 
     //TODO remove or comment out, this is mocking code
```

Comment:
Let's remove this one if we aren't using it at all.... specially given it has a TODO message